packages/hardhat-ledger/test/provider.ts

```diff
@@ -144,6 +144,7 @@ describe("LedgerProvider", () => {
         accounts,
         openTimeout: 1000,
         connectionTimeout: 5432,
+        derivationFunction: (index: number) => `m/44'/60'/${index}'/0/`,
```

Comment:
Could you clarify if there's a need to specify a `derivationFunction` here? It seems like it might not be necessary, but I might be missing something.

Reply:
I agree that it is not necessary, and I have removed it.